weather.py

```diff
@@ -1,72 +1,143 @@
 #!/usr/bin/env python3
 
+from dataclasses import dataclass
+from datetime import datetime
+import dacite
 import json
 import requests
+import sys
```

Comment:
Any pip installable packages should be specified in `requirements.txt`
At the moment you are using requirements.txt as actual app requirement which is incorrect.
See [here](https://learnpython.com/blog/python-requirements-file/)

Reply:
OK. I've generated a proper requirements.txt file vie pip freeze.

Have also installed VSCode with the suggested Python plugin... It's really good.